app/views/course/assessment/assessments/_assessment.html.slim

```diff
@@ -8,8 +8,12 @@
 
   - if current_course.gamified?
     td.table-base-exp = assessment.base_exp
-    - if show_time_bonus_exp?
-      td.table-time-bonus-exp = assessment.time_bonus_exp
+  - if show_bonus_attributes?
+    td.table-time-bonus-exp
+      - if assessment.bonus_end_at.present? && assessment.time_bonus_exp > 0
```

Comment:
If bonus_exp is present and no bonus_end_at,  we should still display bonus_exp ( other than a `-` ).